test/e2e/e2e-skopeo-copy.bats

```diff
@@ -33,7 +33,7 @@ declare -rx E2E_SC_PARAMS_DESTINATION_IMAGE_URL="${E2E_SC_PARAMS_DESTINATION_IMA
     assert_success
 
     # waiting a few seconds before asserting results
-	sleep 30
+	sleep 100
```

Comment:
@Senjuti256 how this resolve verbose issue :thinking:

Reply:
Answered this in slack @savitaashture

Reply:
Okay

```
There wasn’t any error when verbose was  true that means I thought that image is getting copied properly only. So suddenly it came to me that perhaps the copying process is not getting completely over when verbose is false. So I checked by increasing sleep time. For kind cluster 90 was fine but for open shift cluster 100 is required minimum
```

Reply:
Added the test @savitaashture